db/migrate/20180917082341_add_conversion_host_limits.rb

```diff
@@ -0,0 +1,11 @@
+class AddConversionHostLimitsColumns < ActiveRecord::Migration[5.0]
+  def change
+    change_table :conversion_hosts do |t|
+      t.string :limits_max_concurrent_transformations
+      t.string :limits_max_cpu
```

Comment:
Same for these, I think they would read better as `<metric>_limit` or `max_used_<metric>`.

Also, just for my own curiosity, are these metrics on the conversion host or limits on the VMs this host can convert?